index.html

```diff
@@ -0,0 +1,137 @@
+            <div class="menu-icon">
+                <i class="fa-solid fa-bars ms-4 mt-4 mb-4 fa-2x d-md-none"></i>
+            </div>
+            <div class="closeIcon hidden">
+                <i class="fa-solid fa-xmark fa-2xl"></i>
+            </div>
+            <div class="menu-wrapper">
+                <a href="index.html" class="menu-item nav-link nav-item d-md-none">Home</a>
+                <a href="about.html" class="menu-item nav-link nav-item">About</a>
+                <a href="#" class="menu-item nav-item nav-link">Program</a>
+                <a href="#" class="menu-item nav-item nav-link">Join</a>
+                <a href="#" class="menu-item nav-item nav-link">Sponsor</a>
+                <a href="#" class="menu-item nav-item nav-link">News</a>
+                <button type="button" class="menu-button d-none d-md-block">WebDev Campaign</button>
+            </div>
+        </nav>
+        <div class="headline mt-4">
+            <div class="container hl-wrapper">
+                <p>Sharing the knowledge</p>
+                <h1>WebDev Summit 2022</h1>
+                <div class="hl-box">
+                    <p>A celebration to Web Development. 4 days of fast-paced seminars, lectures, exhibitions and many
+                        more activities. Join us in this awesome summit where more than 80 countries come together to share their knowledge on web development</p>
+                </div>
+                <p class="date">2022.12.12(MON) ~ 15(THU)</p>
+                <p class="date-text">@ National Center of New Technologies </p>
+            </div>
+        </div>
+    </header>
+    <main>
+        <section class="main-program pt-4 pb-4">
+            <h2 class="text-center text-white">Main Program</h2>
+            <hr class="section-hr mx-auto mb-4">
+            <div class="program-wrapper container d-flex flex-column flex-md-row">
+                <div class="activity d-flex flex-md-column align-items-center justify-content-between justify-content-md-around">
+                    <img src="img/lecture.png" alt="" class="activity-img">
+                    <h3 class="w-25">Lecture</h3>
+                    <p class="w-50 text-white">Lorem ipsum dolor sit amet consectetur adipisi elit, natus ea id.</p>
+                </div>
+                <div class="activity d-flex flex-md-column align-items-center justify-content-between justify-content-md-around">
+                    <img src="img/exhibition.png" alt="" class="activity-img">
+                    <h3 class="w-25">Exhibition</h3>
+                    <p class="w-50 text-white">Fugiat quaerat tempore repudiandae expedita doloribus, sit consectetur.
+                    </p>
+                </div>
+                <div class="activity d-flex flex-md-column align-items-center justify-content-between justify-content-md-around">
+                    <img src="img/forum.png" alt="" class="activity-img">
```

Comment:
Kindly, the Alt attributes should contain a description of your image, that will greatly help the accessibility of your website.